package.json

```diff
@@ -33,6 +33,8 @@
     "docs:watch": "npm run docs:prepare && gitbook serve",
     "docs:publish": "npm run docs:clean && npm run docs:build && ./scripts/publish-docs",
     "flow": "eslint packages",
+    "todo": "leasot 'packages/**/*.js' --ignore '**/node_modules','**/dist','**/lib'",
+    "todo-staged": "leasot -x --reporter markdown 'packages/**/*.js' --ignore '**/node_modules','**/dist','**/lib' > TODO.md && git add TODO.md",
```

Comment:
This is a strange command. I think this should just goto the lint-staged.config.js